database/migrations/2023_12_01_161507_create_collection_votes_table.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+declare(strict_types=1);
+
+use App\Models\Collection;
+use App\Models\Wallet;
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration
+{
+    public function up(): void
+    {
+        Schema::create('collection_votes', function (Blueprint $table) {
+            $table->id();
+            $table->foreignIdFor(Wallet::class, 'wallet_id')->constrained();
+            $table->foreignIdFor(Collection::class, 'collection_id')->constrained();
+            $table->timestamp('voted_at')->nullable();
+            $table->timestamps();
```

Comment:
I'd not add timestamps as there will be no update and `voted_at` already mimics `created_at`, wdyt?

Reply:
@shahin-hq makes sense